testsuite/features/reposync/srv_enable_sync_products.feature

```diff
@@ -43,6 +43,14 @@ Feature: Be able to list available products and enable them
     And I should get "- sle-product-we15-sp4-pool-x86_64-sled"
     And I should get "Product successfully added"
 
+  @cloud
+  Scenario: Enable "Public Cloud Module 15 SP3 x86_64" for migration on cloud
```

Comment:
Really nitpicky, I know, but the scenario mentions SP3 and the steps enable SP4

Reply:
Nice catch ! I started with SLE15SP3 but after rebase I had the joy to see we switch to sles15SP4